libs/dtm/dtm_recovery.py

```diff
@@ -53,10 +57,14 @@ def __init__(self, access_key=ACCESS_KEY, secret_key=SECRET_KEY):
         self.secret_key = secret_key
         self.ha_obj = HAK8s()
         self.s3t_obj = S3TestLib(access_key=self.access_key, secret_key=self.secret_key)
+        self.setup_type = CMN_CFG["setup_type"]
+        self.system_random = secrets.SystemRandom()
 
+    # pylint: disable-msg=too-many-locals
     # pylint: disable=too-many-arguments
-    def perform_write_op(self, bucket_prefix, object_prefix, no_of_clients, no_of_samples, obj_size,
-                         log_file_prefix, queue, loop=1, created_bucket: list = None):
+    def perform_write_op(self, bucket_prefix, object_prefix, no_of_clients, no_of_samples,
+                         log_file_prefix, queue, obj_size: str = None, loop: int = 1,
+                         created_bucket: list = None, retry: int = None):
```

Comment:
will retry None will work or we need to explicitely mention as 0?

Reply:
None should work, we have added default None in s3bench lib also.